libxrdp/xrdp_rdp.c

```diff
@@ -244,6 +234,48 @@ xrdp_rdp_read_config(struct xrdp_client_info *client_info)
                 g_strncpy(client_info->key_file, value, 1023);
             }
         }
+        else if (g_strcasecmp(item, "rsakeys_ini") == 0)
+        {
+            if (value[0] != '/')
+            {
+                log_message(LOG_LEVEL_WARNING,
+                            "rsakeys.ini file should use absolute path, using "
+                            "default instead: %s", client_info->rsakeys_ini_file);
+            }
+            else
+            {
+                /* use user defined rsakeys.ini */
+                g_strncpy(client_info->rsakeys_ini_file, value, 1023);
+            }
+        }
+        else if (g_strcasecmp(item, "xrdp_keyboard_ini") == 0)
```

Comment:
Is this feature really important? Why would different instances of xrdp need to use different keyboard configuration?